Package.resolved

```diff
@@ -231,16 +248,20 @@
         "repositoryURL": "https://github.com/tree-sitter/tree-sitter-rust.git",
         "state": {
           "branch": "master",
+<<<<<<< HEAD
```

Comment:
One more here

Reply:
How exactly do I do that? 😂

Reply:
Just remove `Package.resolved` file and then in Xcode: File -> Packages -> Resolve Package Versions